delivery_package_gs1_barcode_report/readme/USAGE.rst

```diff
@@ -0,0 +1,4 @@
+#. Create a new transfer with a product with a serial number.
+#. Assign packages to the individual products in the transfer
+#. On the transfer, click Print SSCC barcode label.
```

Comment:
Add: a PDF report will show with GS1 barcodes describing the weight, SSCC code, and the amount of items in the package.